### DISCUSSION TITLE
Is there a way to make dependent dropdowns?

### DISCUSSION THREAD

**Discussion by hassanjrao** [sa-si-dev/virtual-select#154] [Q&A] [answered]
I have two select dropdowns, 2nd dropdown selected option is dependent on first one. So if I select any option from first dropdown, then the respective option in the 2nd dropdown get selected.

**Comment by sa-si-dev** [comment, marked as answer]
@hassanjrao it has to be handled in your project code.

Use change event listener for first drop-down and update the value for second drop-down on first drop-down change.

```js
document.querySelector('#sample-select1').addEventListener('change', function () {
  const firstValue = this.value;
  const secondValue = 'set value for second dropdown based on firstValue';

  document.querySelector('#sample-select2').setValue(secondValue);
});
```
